src/_includes/layouts/blog-post.njk

```diff
@@ -21,7 +21,7 @@ layout: navigation
       {% if image %}
         <div class="to-lg:w-full-breakout">
           <div class="aspect-ratio aspect-ratio--16x9 aspect-ratio--contain bg-grey-500 mb-8 lg:mb-10 xl:mb-12">
```

Comment:
We'll perhaps have to keep an eye on how the blog post image creation goes here. Not sure I'm 100% on enforcing 16:9 w/ contain will result in the desired effect?

Would we  look at doing away with the fixed-ratio container in some circumstances, and instead let the browser try to hold space in the page by giving images `width`/`height` attributes?

Side note: We might also be able to get that dimensions metadata should we explore something like the [11ty Image plugin](https://www.11ty.dev/docs/plugins/image/) in future 👀

Reply:
The 11ty Image plugin looks nice and definitely worth considering. Another option is the [CSS aspect-ratio](https://developer.mozilla.org/en-US/docs/Web/CSS/aspect-ratio) attribute, although it doesn't have support in Safari (at the moment ) or IE. It has an `auto` option which should work for images that don't adhere to a proper ratio.

One to revisit in the future.

Reply:
I've done a little R&D into the 11ty Image plugin but this can be continued on it's own branch as it raises some new questions.

I've removed the `aspect-ratio` from the hero images on the editorial content pages like `blog-post`, `press-releases` etc...